### PR TITLE
Random things

### PR DESCRIPTION
**Pull request by nearnshaw**
Some random minor things

@oscnet please add changes to the translation

### CODE REVIEW

File: _posts/development-guide/2018-01-09-external-3d-models.md [hunk 1]
```diff
@@ -186,6 +186,33 @@ Examples of other valid sizes:
...
+1. Check the current material's settings to see what texture files are being used and how they are configured.
+2. Delete the current material from the mesh.
+
+   ![](/images/media/materials_delete_material.png)
+
+3. Create a new material.
+
+    <img src="/images/media/materials_new_material.png" alt="New default basic material" width="400"/>
+
+   > TIp: If you're using Blender and are on the _Blender Render_ tab, it creates a basic material by default, which is supported by Decentraland.
```
Comment (eordano) [inline _posts/development-guide/2018-01-09-external-3d-models.md]: `Tip`

File: _posts/development-guide/2018-01-09-external-3d-models.md [hunk 2]
```diff
@@ -284,53 +313,53 @@ In a Decentraland scene, you can use `weight` to blend several animations or to
 
 > Tip: Instead of creating your own animations, you can also download generic animations and apply them to your model. For example, for 3D characters with human-like characteristics, you can download free or paid animations from [Mixamo](https://www.mixamo.com/#/).
 
-#### Creating an animation
+#### Create an animation
```
Comment (eordano) [inline _posts/development-guide/2018-01-09-external-3d-models.md]: `How to`?

**Review by eordano — CHANGES_REQUESTED** [review]
Two minor things!

**Comment by nearnshaw** [comment]
@eordano : I just made some adjustments based on your feedback

**Comment by nearnshaw** [comment]
Ahí hice un par de ajustes en base a tus comments,
Gracias!

**Review by eordano — APPROVED** [review]

**Pull request merged**